multi-ai-cli/backends/grok_web.py

```
@@ -0,0 +1,108 @@
+                            return response_data["modelResponse"]["message"]
+                            
+                        # Handle streaming tokens
+                        token = response_data.get("token", "")
+                        if token:
+                            full_response += token
+                    except:
+                        continue
+            
+            return full_response.strip() if full_response else "[No content returned from Grok]"
+
+        except Exception as e:
```

Pylint (4.0.6)

[warning] Catching too general exception Exception

(W0718)